packages/fx-core/package.json

```diff
@@ -104,7 +104,7 @@
     "@feathersjs/hooks": "^0.6.5",
     "@microsoft/dev-tunnels-contracts": "1.1.9",
     "@microsoft/dev-tunnels-management": "1.1.9",
-    "@microsoft/kiota": "1.25.1",
+    "@microsoft/kiota": "1.25.1-preview.202504170001",
```

Comment:
Will this package have stable version? A version range might be helpful.

Reply:
This feature is under featureflag, and kiota npm package is not fully ready now and may break for range version, so we use preview here.